File: yongkeun/1week/두수의합.js
function solution(a, b) {
  var answer = String(BigInt(a) + BigInt(b)); //a, b를 Number로 변환하면 숫자가 너무 커서 뒤에가 짤림. 그러므로 더 큰 숫자를 담을 수 있는 BigInt를 이용하여 연산 후, String으로 바꿔서 answer에 할당
  return answer;
}

File: minwook/1주차/두_수의_합.java
import java.math.BigInteger;

class Solution {
    public String solution(String a, String b) {
        BigInteger num1 = new BigInteger(a); // 자릿수를 초과하는 큰 정수를 처리하기 위한 BigInteger 사용
        BigInteger num2 = new BigInteger(b);
        String answer = num1.add(num2) + ""; // add 함수로 덧셈 ( 사칙연산 사용 시 에러가 발생 )
        return answer;
    }
}